src/framework/fixture.ts

```diff
@@ -66,12 +66,12 @@ export class Fixture {
 
   private expectErrorValue(expectedName: string, ex: unknown, m: string): void {
     if (!(ex instanceof Error)) {
-      this.fail('THREW NON-ERROR');
+      this.fail('THREW non-error value, of type ' + typeof ex);
```

Comment:
This isn't silencing a warning, but it verbosely explains a specific error in Safari.